src/tasks/BaseNTETask.py

```
@@ -43,6 +44,77 @@ def __init__(self, *args, **kwargs):
         self._last_interval_action_time = {}
         self._action_interval_lock = threading.Lock()
 
+    def log_info(self, message, notify=False):
+        """覆盖 log_info 以累积保存日志"""
+        self.logger.info(message)
+        self._add_log_to_info("Log", message)
+        if notify:
+            self.notification(message, tray=True)
+
+    def _add_log_to_info(self, prefix, message):
+        """将日志添加到 info 字典，编号持续递增，只保留最新 10 条"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)